app/views/users/index.html.erb

```diff
@@ -1 +1,7 @@
-<h1>Lists of user </h1>
\ No newline at end of file
+<ul>
+<% @users.each do |user| %>
+<li><%=user.name%><span>Number of 
+Posts: <%=user.posts_counter%></span></li>
+
+<%end%>
```

Comment:
- [ ] [optional] @Lul-Abdifan it would have been better if you used a partial view to display the users instead of using @users.each , example code

 ```
   app/views/users/_user.html.erb

     #   HTML Code Here displaying
     #   user.photo, user.name and user.posts_counter

    app/views/users/index.html.erb

    <%= render @users %>
```